### PR TITLE
Vectra AI Solutions

### PR DESCRIPTION
**Pull request by danymello**
Moving Vectra Content as Solution  

- Data Connectors for Detect and Stream
- Parser for Stream
- Workbook for Detect
- Analytic Rules for Detect

### CODE REVIEW

File: Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
+{
+    "id": "AIVectraStream",
+    "title": "AI Vectra Stream",
+    "publisher": "Vectra AI",
+    "descriptionMarkdown": "The AI Vectra Stream connector allows to send Network Metadata colelcted by Vectra Sensors accross the Network and Cloud to Azure Sentinel", 
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Install Kusto functions prior creating new queries or workbook.",
```
Comment (v-admahe) [inline Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json:6]: Parser link should be the part of additional requirement banner description. Please add the parser link in this format https://aka.ms/sentinel-Parser-AIVectraStream.

Thanks,
Aditi Maheshwari

File: Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            {
+                "name": "Vectra AI Brain",
+                "description": "must be configured to export Stream metadata in JSON"
+            }
+        ]
+    },
+    "instructionSteps": [
+        { 
+            "title": "", 
+            "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected **VectraStream** which is deployed with the Azure Sentinel Solution.",
```
Comment (v-admahe) [inline Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json:83]: Parser link should be the part of this note. Please add the parser link in this format https://aka.ms/sentinel-Parser-AIVectraStream.

**Review by v-admahe — CHANGES_REQUESTED** [review]

**Comment by v-admahe** [comment]
Please add the sample data for VectraStream_CL in the json format.

Thanks,
Aditi Maheshwari

**Comment by v-admahe** [comment]
@danymello The connector AIVectraDetect.json and workbook AIVectraDetectWorkbook.json are already a part of dataconnector and workbook so is there any update on them or you are moving them into solutions. In case moving, than files should be moved not added to the solution.

Thanks,
Aditi Maheshwari

File: Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json (hunk 1 above)
Comment (danymello) [reply Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json:6]: @v-admahe, how those links are built? Any pointer?

**Comment by danymello** [comment]
> @danymello The connector AIVectraDetect.json and workbook AIVectraDetectWorkbook.json are already a part of dataconnector and workbook so is there any update on them or you are moving them into solutions. In case moving, than files should be moved not added to the solution.
> 
> Thanks,
> Aditi Maheshwari

They are the same (last PR for the worbook has benn merged: https://github.com/Azure/Azure-Sentinel/pull/2301). Should I delete them from the Connector and Workbook folders and have them only in Solution folder instead?

**Comment by v-jayakal** [comment]
@danymello , for now, since connector and workbook are already deployed, we will not consider these under solution. The initial PR with detections https://github.com/Azure/Azure-Sentinel/pull/2329 will be reviewed and merged into master under Sentinel\Detections. We will consider AlVectra solution as part of our migration roadmap which is currently in planning stage as we have to consider all aspects about already deployed existing components. (For example, if parser is already created in Legacy category, adding and deploying it via solution will result in duplication etc).
My apologies for the confusion caused here by the ask of the solution. We will close this PR for now. 
I already sent this information in email. Please let us know if you need any more information. Thanks.

**Pull request closed without merging**